### PR TITLE
Grammar pass

### PR DESCRIPTION
**Pull request by clee-top**
While reading all the documentation and going through the steps for learning purposes fixed up some of the grammar.

### CODE REVIEW

File: docs/github-actions.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ in the workflow.
           sonobuoy version
 ```
 
-This is a working verison of it but there were many itterations before I got the tar output correct
+This is a working venison of it but there were many iterations before I got the tar output correct
```
Comment (sekka1) [inline docs/github-actions.md]: This should be "version"?

Comment (clee-top) [reply docs/github-actions.md]: Haha! You're right, silly me.

Comment (sekka1) [reply docs/github-actions.md]: Thanks =)

**Comment by sekka1** [comment]
Thanks for the grammar fixes @clee-top

**Pull request merged**